### DISCUSSION TITLE
A method similar to OnPlay, as in DoTween.

### DISCUSSION THREAD

**Discussion by Keshbel** [KyryloKuzyk/PrimeTween#99] [General] [answered]
What is really missing is the OnPlay analogue method, as in DoTween. To perform some actions after continuing to play the tween.

**Comment by KyryloKuzyk** [comment, marked as answer]
You can just execute the code manually before you create or unpause the animation. Will that solve your feature request?
More info here: https://github.com/KyryloKuzyk/PrimeTween/discussions/90#discussioncomment-10365775
